tracing/tracer.go

```diff
@@ -41,8 +41,9 @@ func InitGlobalTracer(
 	var reporter *jaegerReporter
 	if cfg == nil || !cfg.Disabled {
 		cfg = loadAppConfig(cfg, logger)
+		// TODO: Change to use the right stats reporter
 		reporter = &jaegerReporter{
-			reporter: scope.Reporter(),
+			reporter: tally.NullStatsReporter,
```

Comment:
oh, is this not possible anymore? are you going to address that in this PR or a future one?